pkg/controller/direct/datacatalog/entry_fuzzer.go

```diff
@@ -66,11 +66,12 @@ func dataCatalogEntryFuzzer() fuzztesting.KRMFuzzer {
 	f.StatusFields.Insert(".usage_signal")
 	f.StatusFields.Insert(".data_source")
 	f.StatusFields.Insert(".personal_details")
+	f.StatusFields.Insert(".source_system_timestamps.expire_time")
 
 	f.UnimplementedFields.Insert(".name") // special field
 	f.UnimplementedFields.Insert(".integrated_system")
-	f.UnimplementedFields.Insert(".source_system_timestamps.expire_time") // Status field
-	f.UnimplementedFields.Insert(".schema.columns")                       // Status field
+	f.UnimplementedFields.Insert(".linked_resource") // spec field
```

Comment:
Not sure I follow. Why this field is in both `SpecFields` and `UnimplementedFields`?

Reply:
This field was removed from spec. But the fuzzer was testing for it. not sure why. So added it here.

Reply:
removed from both